bundle/manifests/operator.ibm.com_commonservices.yaml

```diff
@@ -56,6 +56,7 @@ spec:
                         type: boolean
                     type: object
                 type: object
+                x-kubernetes-preserve-unknown-fields: true
```

Comment:
Is this clause applying to `spec.features` instead of `spec`?

Reply:
Oh nice catch, I put it in wrong place thanks